src/Kunstmaan/MediaBundle/Command/CleanDeletedMediaCommand.php

```diff
@@ -30,6 +65,11 @@ protected function configure()
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        if (null === $this->em) {
```

Comment:
I would do this in the constructor. Every other thing we deprecated is done in the constructur.

Reply:
We can't access the container yet in the constructor as this will be set later. Using the container in the constructor throws this error
```
[LogicException]
  The container cannot be retrieved as the application instance is not yet set.
```

Reply:
Correct 😉